### DISCUSSION TITLE
Consider adding Pi-hole usage in Wiki section as well. (Pi-hole kullanımı da Wiki bölümüne eklenebilir.)

### DISCUSSION THREAD

**Discussion by DellendinMi** [symbuzzer/Turkish-Ad-Hosts#18] [Genel sohbet / General chat]
Not so essential, but you may consider adding Pi-hole usage as well, in Wiki section.
**Nice work by the way, thanks for the effort.**

Çok gerekli değil ancak Pi-hole kullanımını da Wiki bölümüne eklemeyi düşünebilirsiniz.
**Güzel çalışma bu arada, emeğiniz için teşekkürler.**

![symbuzzer Turkish-Ad-Hosts 006](https://github.com/symbuzzer/Turkish-Ad-Hosts/assets/67484569/d1861173-52c8-4526-9836-ca7696ffd1db)

**Comment by symbuzzer** [comment]
Merhaba, ben teşekkür ederim, beğenmenize sevindim.
Pi-hole'u hiç kullanmadım. Adım adım nasıl kurulabileceğini anlatabilirseniz eklerim tabi seve seve

**Reply by symbuzzer** [reply]
@DellendinMi, yol haritası konusunda yardımcı olmanız mümkünse wiki'ye ekleyelim?

**Reply by DellendinMi** [reply]
Pardon, Github ve benzerlerinde pek aktif olmadığım için mesajlarınızı geç gördüm.

Pi-hole kurulumu başlı başına ayrı bir mesele olduğu için onu kastetmiyorsunuzdur sanırım. Pi-hole'ü halihazırda kullanan kişiler ise sizin hazırladığınız hosts dosyasının adresini Pi-hole arayüzündeki listelere Adlists-->Adlist group management-->Add a new adlist-->Address: kutusuna girerek ve "Add" kutusuna tıklayarak ekleyebilirler. Aynı diğer listelerin de eklenebildiği gibi. Bu zaten ilk mesajıma eklediğim Pi-hole arayüzü ekran görüntüsünde görülebilir.

Bu şekilde sizin adres de aynı sayfada List of adlists-->Addresses bölümünde görülmeye başlanacak. Listenin hemen indirilip kullanılmaya başlanması için Raspberry Pi cihazı (veya sanal makina) üzerinden veya SSH ile bağlanarak "pihole -g" komutunu girmek gereklidir. Ancak daha kolay olarak aynı sayfadaki mavi "online" linkine tıklayarak kısa yoldan Tools-->Update Gravity-->Update Gravity (list of blocked domains) sayfasına ulaşarak Update butonuna tıklamak yeterlidir. Cihaz tüm listeleri kontrol eder ve varsa yeni girişleri de ekleyerek database'i günceller (Success yazısı çıkana dek bekleyiniz).

Bunları madde madde alt alta yazmam daha anlaşılır olabilir:

**1)** Pi-hole admin web arayüzünde "Adlists" sayfasını açın.
**2)** Bu sayfadaki "Adlist group management-->Add a new adlist-->Address:" kutusuna Turkish Ad Hosts listesinin adresini yani "https://raw.githubusercontent.com/symbuzzer/Turkish-Ad-Hosts/main/hosts" u girin ve "Add" kutusuna tıklayarak ekleyin. (İsteğe bağlı olarak "Comment:" böümüne listeyle ilgili bir açıklama da girilebilir.)
**3)** Adresin aynı sayfada "List of adlists-->Addresses" bölümünde görüldüğünden emin olun.
**4)** Listenin hemen indirilip kullanılmaya başlanması için;
  a) Raspberry Pi cihazı (veya sanal makina) üzerinden veya SSH ile bağlanarak "pihole -g" komutunu girin, veya,
  b) Daha kolay olarak aynı sayfadaki mavi "online" linkine tıklayarak kısa yoldan "Tools-->Update Gravity-->Update Gravity (list of blocked domains)" sayfasına ulaşarak Update butonuna tıklayın, veya,
  c) Direkt "Tools-->Update Gravity-->Update Gravity (list of blocked domains)" sayfasına ulaşarak Update butonuna tıklayın.
**5)** Cihaz tüm listeleri kontrol eder ve varsa yeni girişleri de ekleyerek database'i günceller. "Success!" yazısı çıkana dek bekleyiniz!

![Pi-hole - raspberrypi Update Gravity 01](https://github.com/symbuzzer/Turkish-Ad-Hosts/assets/67484569/257ece03-9f0e-4771-b9be-1b64ee6764cd)

![Pi-hole - raspberrypi Update Gravity](https://github.com/symbuzzer/Turkish-Ad-Hosts/assets/67484569/96776af6-a726-4c61-87f5-95a3cf18d074)

Sanırım bu şekilde özetlenebilir. Zaten Pi-hole kullanıcılarının çoğu bu listelerin eklenmesini biliyordur muhtemelen. Ben de deneyimli bir kullanıcı olmasam da bir süredir Raspberry Pi B Rev2 512MB Ram modeli üzerinde Pi-Hole'ü sorunsuz kullanıyorum.

Pi-Hole genelde Raspberry Pi cihazlar üzerinde kullanılsa da, sanal makinalarda veya aşağıda en alttaki linkte anlatıldığı gibi yedekte duran ROOTlu(!) android telefonlar/tabletler üzerine de kurulup kullanılabiliyor, isteyenler detayları araştırabilirler. Benim de yararlandığım bazı faydalı olabilecek linkler:

* https://docs.pi-hole.net/
* https://ozkancelik.com/pi-hole-kurulumu/
* https://www.technopat.net/sosyal/konu/pi-hole-kurulumu-modeme-bagli-tuem-cihazlardaki-reklamlari-engellemek.1469960/
* https://forum.donanimhaber.com/pi-hole-kurulum-rehberi-2021-raspberry-pi-zero--147439130
* https://github.com/DesktopECHO/Pi-hole-for-Android

Umarım bir nebze faydası olur. İyi günler.

**Reply by symbuzzer** [reply]
İlk fırsata wikiye ekleyecem hocam, elinize sağlık, çok teşekkür ederim
